rally/track.json

```diff
@@ -0,0 +1,55 @@
+{% import "rally.helpers" as rally with context %}
+{
+  "version": 2,
+  "description": "APM benchmark for Rally",
+  "indices": [
+    {
+      "name": "apm-6.4.0-error",
+      "body": "index.json",
+      "types": [ "type" ]
+    },
+    {
+      "name": "apm-6.4.0-transaction",
+      "body": "index.json",
+      "types": [ "type" ]
+    },
+    {
+      "name": "apm-6.4.0-span",
+      "body": "index.json",
+      "types": [ "type" ]
+    }
+  ],
+  "corpora": [
+    {
+      "name": "apm-events",
+      "target-type": "type",
+      "documents": [
+        {
+          "target-index": "apm-6.4.0-error",
+          "source-file": "documents/error.json",
+          "document-count": 224250,
+          "uncompressed-bytes": 5366030615
+        },
+        {
+          "target-index": "apm-6.4.0-transaction",
+          "source-file": "documents/transaction.json",
+          "document-count": 1224774,
+          "uncompressed-bytes": 2224034492
+        },
+        {
+          "target-index": "apm-6.4.0-span",
+          "source-file": "documents/span.json",
+          "document-count": 2225454,
+          "uncompressed-bytes": 10897616683
+        }
+      ]
+    }
+  ],
+  "operations": [
```

Comment:
You could probably inline operations and challenges which would make it a bit simpler to read. Then you can get rid of the first line (`{% import "rally.helpers" as rally with context %}`) and the `rally.collect` calls. You can see a [simple example track](https://esrally.readthedocs.io/en/stable/adding_tracks.html#example-track) in our docs).

Reply:
I think it will be easier to avoid code duplication in the future, e.g. when adding querying, when we don't inline.